docs/testing.md

```diff
@@ -55,6 +55,15 @@ yarn test:watch
 
 The tests will perform an initial run and then re-execute each time a project file is changed.
 
+### Running the Jest front-end integration tests
+
+The integration tests for the UI are run with [Jest] and [Puppeteer].
+
+To run the tests:
+
+- If you haven't already done so, install local dependencies by running `yarn install` from the project root.
+- Then run `yarn test:integration` to execute the tests.
```

Comment:
This reads like a duplicate of the above section. Can you combine these two sections and mention that tests are written with react testing library and that Polly can be used to mock APIs (maybe some info about the best time to use it over using fetch mock?).

Reply:
Thank you, that's much clearer.